app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
```

Comment:
Code Style: According to the checklist, it's better to use parentheses for multiple imports. This makes the code more readable.